environments/development/common/rds.tf

```diff
@@ -15,11 +15,16 @@ module "postgres" {
 
   allocated_storage     = 20
   max_allocated_storage = 40
+  security_group_ids    = [data.aws_security_group.be_to_rds_security_group.id]
 
   region      = var.region
   environment = var.environment
 }
 
+data "aws_security_group" "be_to_rds_security_group" {
+  id = module.alb-security-group.be_to_rds_security_group_id
+}
```

Comment:
```suggestion
```

No need for the data source to fetch this, it's already an output from the module